portfolio/src/main/webapp/style.css

```diff
@@ -55,20 +96,145 @@ body {
     width: auto;
 }
 
+/*************** loader ***************/
+.loader {
+  border: 16px solid #ffffff; /* white */
+  border-top: 16px solid #3498db; /* Blue */
+  border-radius: 50%;
+  width: 120px;
+  height: 120px;
+  animation: spin 2s linear infinite;
+  
+  position: absolute;
+  top: 40%;
+  left: 50%;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+
+
+
+/*************** content ***************/
+
+#contentspacer{
```

Comment:
There should be a space before the opening curly bracket {

Some guidance on CSS spacing: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation